src/ucp/proto/proto_am.inl

```diff
@@ -413,10 +413,10 @@ ucp_proto_get_zcopy_threshold(const ucp_request_t *req,
                               const ucp_ep_msg_config_t *msg_config,
                               size_t count, size_t max_zcopy)
 {
-    ucp_worker_h     worker;
+    ucp_worker_h worker;
```

Comment:
afair, we have alignment by vars in many places..

Reply:
yes, you're right. but it's more common to not have alignment by vars